examples/own_dataset/train.py

```diff
@@ -50,6 +54,26 @@ def __call__(self, atoms, adjs):
         return x
 
 
+class ScaledAbsError(object):
+
+    def __init__(self, scale='standardize', ss=None):
+        self.scale = scale
+        self.ss = ss
```

Comment:
Could you reconsider the attribute name so that users can understand its functionality from its name?

Reply:
changed to scaler